pkgs/applications/misc/stag/default.nix

```diff
@@ -11,14 +11,21 @@ stdenv.mkDerivation {
     sha256 = "1yrzjhcwrxrxq5jj695wvpgb0pz047m88yq5n5ymkcw5qr78fy1v";
```

Comment:
rev should be chaned to `rev = "v${version}";`

Reply:
done